components/CommonLinks/CommonLinks.module.css

```diff
@@ -0,0 +1,49 @@
+.CommonLinks__container {
+    position: fixed;
+    display: flex;
+    gap: 12px;
+    background-color: rgb(239 240 241);
+    border-radius: 10px 10px 0 0;
+    min-width: 376px;
+    margin-left: 15px;
+    bottom: 0;
+}
+
+.CommonLinks__icon {
+    margin: 16px 0 16px 16px;
+}
+
+.CommonLinks__link {
+    text-decoration: none;
+    font-weight: 400;
+    box-shadow: none;
+    color: black;
+    vertical-align: middle;
+    text-decoration: none;
+    border-radius: 10px;
+    transition: 0.2s;
+    padding: 15px;
+    font-size: 1rem;
```

Comment:
Let's use `px` and `rem`. Our application has become much more component-based to depend on global values

Reply:
We can put it in a separate Issue